invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/files_integrity_report/email/files_integrity_report.html

```diff
@@ -22,10 +21,10 @@
 {{ "Last check date: %s" | format(entry.file.last_check_at) }}
 {{ "Last check FAILED with result: %s" | format(entry.file.last_check) }}
 {%- if 'record' in entry %}
-{{ "Record: %s/records/%s" | format(BASE_URL, entry.record.id) }}
+{{ "Record: %s" | format(invenio_url_for("invenio_app_rdm_records.record_detail", pid_value=entry.record.id)) }}
```

Comment:
Other 2 of 2 main integration: generate links in email templates, so we don't have to cobble together hardcoded urls.